project/SettingsPublish.scala

```diff
@@ -22,13 +22,15 @@ trait SettingsPublish {
 
   lazy val publishSnapshot = taskKey[Unit]("Publish only if the version is a SNAPSHOT")
 
+  lazy val gpgFolder = sys.env.getOrElse("GPG_FOLDER", ".")
+
   lazy val publishSettings = Seq(
     publishSnapshot := { if(isSnapshot.value) publishSigned.value },
     publishMavenStyle := true,
     // https://github.com/sbt/sbt-pgp/issues/80
-    com.typesafe.sbt.SbtPgp.autoImport.pgpPassphrase := Some(sys.env("GPG_PASSPHRASE").toCharArray),
-    com.typesafe.sbt.SbtPgp.autoImport.pgpPublicRing := file("local.pubring.asc"),
-    com.typesafe.sbt.SbtPgp.autoImport.pgpSecretRing := file("local.secring.asc"),
+    com.typesafe.sbt.SbtPgp.autoImport.pgpPassphrase := Some(sys.env.getOrElse("GPG_PASSPHRASE", "").toCharArray),
```

Comment:
Do we want to provide a default value here or simply throw an exception if no GPG_PASSPHRASE is provided?